ci/do_ci.sh

```diff
@@ -16,7 +16,17 @@ SRCDIR="${PWD}"
 . "$(dirname "$0")"/build_setup.sh $build_setup_args
 cd "${SRCDIR}"
 
+if [[ "${ENVOY_BUILD_ARCH}" == "x86_64" ]]; then
+  BUILD_ARCH_DIR="/linux/amd64"
+elif [[ "${ENVOY_BUILD_ARCH}" == "aarch64" ]]; then
+  BUILD_ARCH_DIR="/linux/arm64"
+else
+  echo "Unknown architecture for building: '${ENVOY_BUILD_ARCH}'"
+  exit 1;
```

Comment:
There are folks building on ppc64le / s390x, Shouldn't fail hard here. Just fall back to `linux/${ENVOY_BUILD_ARCH}`?

Reply:
NP. I will revise my patch.